src/Evolution/Systems/Cce/Actions/InitializeWorldtubeBoundary.hpp

```diff
@@ -45,13 +45,16 @@ namespace Actions {
 
 namespace detail {
 template <typename Initializer, typename ManagerTags,
-          typename BoundaryCommunicationTagsList>
+          typename BoundaryCommunicationTagsList1,
+          typename... BoundaryCommunicationTagsList2>
```

Comment:
With just this commit, this actually adds a Variables tag with an empty list since there are no extra tag lists yet (which is not something we want). I'd recommend something like

```cpp
template <typename Initializer, typename ManagerTags,
          typename... BoundaryCommunicationTagsList> {
  // other stuff

  simple_tags = tmpl::list<::Tags::Variables<BoundaryCommunicationTagsList>...>;

  // inside apply function
    Initialization::mutate_assign<simple_tags>(make_not_null(&box),
                     Variables<BoundaryCommunicationTagsList>{
        Spectral::Swsh::number_of_swsh_collocation_points(l_max)}...);
};
```

Not that we'll likely need more than 2 lists, but this also avoids the annoying `1` and `2` suffixes.

Reply:
Good idea!